.eslintrc.json

```diff
@@ -1,37 +1,48 @@
 {
-    "extends": "google",
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-      "ecmaVersion": "2017",
-      "sourceType": "module"
-    },
-    "plugins": [
-      "@typescript-eslint"
-    ],
-    "rules": {
-      "max-len": ["error", {
+  "extends": ["plugin:prettier/recommended", "google"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "2017",
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "indent": "off",
+    "quote-props": "off",
+    "max-len": [
+      "error",
+      {
         "ignoreStrings": true,
-        "ignoreTemplateLiterals": true
-      }],
-      "no-unused-vars": "off",
-      "spaced-comment": "off",
-      "comma-dangle": ["error", "never"],
-      "quotes": ["error", "single", {
+        "ignoreTemplateLiterals": true,
+        "ignoreUrls": true
+      }
+    ],
+    "no-unused-vars": "off",
+    "spaced-comment": "off",
+    "comma-dangle": ["error", "never"],
+    "quotes": [
+      "error",
+      "single",
+      {
         "avoidEscape": true,
-        "allowTemplateLiterals": true
-      }],
-      "@typescript-eslint/class-name-casing": "error",
-      "@typescript-eslint/explicit-function-return-type": "error",
-      "@typescript-eslint/explicit-member-accessibility": "error",
-      "@typescript-eslint/member-delimiter-style": "error",
-      "@typescript-eslint/member-naming": ["error", {
+        "allowTemplateLiterals": false
```

Comment:
shouldn't this be true? we should use template literals for any interpolation (maybe we don't yet have any?)